### PR TITLE
Newsletter-6: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

File: _posts/zh/newsletters/2018-07-31-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+- **<!--fees-still-low-->费用仍然低**：在周日结束的 2016 区块重新目标周期中，哈希率增加了超过14%，使区块间平均时间为 8 分 41 秒。这有助于吸收过去一周价格投机带来的交易量增加。紧随难度重新目标之后，区块间平均时间恢复为 10 分钟。
+
+  随着我们从正常的周末交易低迷过渡到新的一周，估计的交易费用可能会迅速增加。我们建议在靠近周末时再发送大量低费用交易（如合并），那时交易量开始再次下降。
+
+## 实地报告：Xapo 合并 400 万个 UTXO
+
+*由 [Xapo][] 的 Bitcoin Core 开发人员 [Anthony Towns](https://twitter.com/ajtowns) 编写*
+
+{% include articles/towns-xapo-consolidation.md hlevel='###' %}
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2018-07-31-newsletter.md:26]: @bittomhan I think it makes sense to also translate this field report from AJ into Chinese as well.

Here is a Chinese translation of the field report in newsletter 277 which could be helpful to reference:
https://github.com/bitcoinops/bitcoinops.github.io/pull/1408/files

What do you think?

Comment (bittomhan) [reply _posts/zh/newsletters/2018-07-31-newsletter.md:26]: I'm glad to have been reminded, as I previously hadn't realized that the content of these articles also needed to be translated into Chinese. I have now translated and submitted a PR. After the PR for the xapo article is approved, I will add the corresponding file index path to it. Thank you very much!

Comment (bitschmidty) [reply _posts/zh/newsletters/2018-07-31-newsletter.md:26]: This needs a reference to the zh version instead

**Review by bitschmidty — CHANGES_REQUESTED** [review]
Youll need an entry in the post folder similar to this PR (https://github.com/bitcoinops/bitcoinops.github.io/pull/1408/files#diff-20c6ce237d6117cfcfb7b9435b071fb60f04101f4e155f20da8334068039cec6)

and its "_posts/zh/2023-11-15-wizardsardine-miniscript.md" file.

This is what also allows the article to be in its own blog post (like this: https://bitcoinops.org/zh/wizardsardine-miniscript/)

**Pull request closed without merging**